beater/server_test.go

```diff
@@ -241,13 +269,12 @@ var testData = func() []byte {
 	return d
 }()
 
-func withSSL(t *testing.T, domain string) *SSLConfig {
-	cert := path.Join(tmpCertPath, t.Name()+".crt")
-	key := path.Join(tmpCertPath, t.Name()+".key")
-	t.Log("generating certificate in ", cert)
-	httpscerts.Generate(cert, key, domain)
+func withSSL(t *testing.T, domain string, passwordKey string) *SSLConfig {
+	name := path.Join(tmpCertPath, t.Name())
+	t.Log("generating certificate in ", name)
+	transptest.GenCertForTestingPurpose(t, domain, name, passwordKey)
 
-	return &SSLConfig{Cert: cert, PrivateKey: key}
+	return &SSLConfig{Enabled: newTrue(), Certificate: outputs.CertificateConfig{Certificate: name + ".pem", Key: name + ".key", Passphrase: passwordKey}}
```

Comment:
since this is the only usage of `newTrue()`, a local `truthy` var like elsewhere would be preferred.